ignite/services/chain/chain.go

```diff
@@ -185,6 +193,24 @@ func New(path string, options ...Option) (*Chain, error) {
 	return c, nil
 }
 
+func NewChainWithHomeFlags(cmd *cobra.Command, chainOption ...Option) (*Chain, error) {
```

Comment:
Maybe the name `Chain` could be removed from the function name:

```suggestion
func NewWithHomeFlags(cmd *cobra.Command, chainOption ...Option) (*Chain, error) {
```